ros2pkg/ros2pkg/api/create_api.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
I would suggest to call the file just `create` since `api` is already part of the namespace.